tap_zoom/client.py

```diff
@@ -67,8 +67,11 @@ def refresh_access_token(self):
                 'grant_type': 'refresh_token'
             })
 
-        self.__access_token = data['access_token']
-        self.__refresh_token = data['refresh_token']
+        try:
+            self.__access_token = data['access_token']
+            self.__refresh_token = data['refresh_token']
+        except:
+            LOGGER.error("Access token or Refresh token cannot be empty. Please check credentials and try again.")
```

Comment:
Instead of just printing this, can you log the response we got from the request too? Something like:

```
LOGGER.error(f"Failed to refresh OAuth tokens, response from Zoom API = [{data}]")
```